lib/messageUtils.ts

```diff
@@ -48,11 +52,16 @@ export function getSurfaceDataForTemplate(template: string): SurfaceData {
       surface: "Protections Dropdown Panel",
       tagColor: "bg-blue-500",
     },
+    rtamo: {
+      surface: "Return to AMO",
+      tagColor: "bg-violet-200",
+    },
     toast_notification: {
       surface: "Toast Notification",
       tagColor: "bg-indigo-400",
     },
     toolbar_badge: { surface: "Toolbar Badge", tagColor: "bg-purple-400" },
+    "toolbar-badge": { surface: "Toolbar Badge", tagColor: "bg-purple-400" },
```

Comment:
This was needed in the original spike because the Looker data uses "toolbar-badge", but now I notice it's not necessary since we are prioritizing the asrouter data which uses "toolbar_badge".

Reply:
I'd guess we're going to need both until we start reading JSON from remote settings also.  Add an XXX comment about this?